course_content/notebooks/numpy_intro.ipynb

```diff
@@ -126,21 +178,40 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "### Calculation with Broadcasting"
+    "### Fast Calculations\n",
+    "\n",
+    "\n",
+    "NumPy provides routines for fast operations on arrays, including mathematical, logical, shape manipulation, sorting, selecting, I/O, discrete Fourier transforms, basic linear algebra, basic statistical operations, random simulation and much more.\n",
+    "\n",
+    "It is a lot faster than Python alone for numerical computing tasks.\n",
+    "\n",
+    "Element-by-element operations are the “default mode” when an ndarray is involved, but the element-by-element operation is speedily executed by pre-compiled C code. "
```

Comment:
It's funny, to me the use of 'speedily' here really increases the informality of this sentence...